Automattic/wp-calypso
# Domains: Make domain name a link in domain card

#### Changes proposed in this Pull Request

This PR makes the domain name a link in the domain card, and adds an external link icon to make that more obvious. This change affects all kinds of domains: registered, mapped, subdomains, etc.

#### Testing instructions

- Go to your site's domains, and click on a domain
- You should see the external link icon next to the domain name
- Clicking the domain name should open the site the domain points to in a new tab

<img width="728" alt="Screenshot 2020-05-22 at 6 12 46 PM" src="https://user-images.githubusercontent.com/13062352/82688035-7b6cbb80-9c58-11ea-8e70-acb4647968b8.png">
